psi4/src/psi4/libfunctional/superfunctional.cc

```diff
@@ -662,7 +662,66 @@ void SuperFunctional::allocate() {
     }
 }
 std::map<std::string, SharedVector>& SuperFunctional::compute_functional(
-    const std::map<std::string, SharedVector>& vals, int npoints) {
+    const std::map<std::string, SharedVector>& vals, int npoints, bool singlet) {
+    if (!singlet && is_unpolarized() and deriv_ == 2) {
+        // We want triplet spin integration for the second derivatives.
+
+        // First, make a UKS version of this functional.
+        auto UKS = build_polarized();
+
+        // Now compute the UKS version of this functional. Naturally, the input values will need to be UKS-ified first.
+        std::map<std::string, SharedVector> UKS_vals;
+        if (true) {
```

Comment:
Is there a reason for couple of if(true)s that I see?

Reply:
This convention traces back to [the LibXC interface](https://github.com/psi4/psi4/blob/master/psi4/src/psi4/libfunctional/LibXCfunctional.cc#L407) but is fundamentally so the LDA block of code "looks like" the GGA and eventually meta blocks. I can also check that "RHO_A" is in the dictionary, but that should never ever be false, so checking it as if it _could_ be false would be misleading.

Reply:
Oh okay, how odd. That's fine. I didn't know the history of it.